core/internal/httpserver/prometheus.go

```diff
@@ -77,19 +76,18 @@ func (hc *Coordinator) handlePrometheusMetrics() http.HandlerFunc {
 				consumerStatusGauge.With(labels).Set(float64(consumerStatus.Status))
 
 				for _, partition := range consumerStatus.Partitions {
-					if partition.Complete < 1.0 {
-						continue
-					}
-
 					labels := map[string]string{
 						"cluster":        cluster,
 						"consumer_group": consumer,
 						"topic":          partition.Topic,
 						"partition":      strconv.FormatInt(int64(partition.Partition), 10),
 					}
 
-					consumerPartitionCurrentOffset.With(labels).Set(float64(partition.End.Offset))
 					consumerPartitionLagGauge.With(labels).Set(float64(partition.CurrentLag))
+
+					if partition.Complete >= 1.0 {
```

Comment:
could `Complete` be greather than 1, or is this only a percentage and we could use here `== 1.0`?